testsuite/tests/basic-more/opaque_prim.ml

```diff
@@ -0,0 +1,8 @@
+
+let f x = opaque x
+
+let () =
+  assert(f f == f);
+  assert(opaque 1 = 1);
+  assert(opaque 1. = 1.)
+
```

Comment:
This tests that  it's the identity, but it doesn't seem to test that it is indeed opaque, right ?